lib/hammer_cli_katello/activation_key.rb

```diff
@@ -56,41 +56,63 @@ def request_params
         field :id, _("Id")
         field :description, _("Description"), Fields::Field, :hide_blank => true
         field :format_consumed, _("Host Limit")
-        field :auto_attach, _("Auto Attach")
+        field :multi_content_view_environment, _("Multi Content View Environment"), Fields::Boolean
         field :release_version, _("Release Version"), Fields::Field, :hide_blank => true
+        field :content_view_environment_labels, _("Content view environments"), Fields::Field
 
-        from :environment do
-          field :name, _("Lifecycle Environment")
+        collection :organization, _("Organization") do
+          field :id, _("Id"), Fields::Field, :hide_blank => true
+          field :name, _("Name"), Fields::Field, :hide_blank => true
         end
-        from :content_view do
-          field :name, _("Content View")
+
+        collection :content_view_environments, _('Content view environments') do
```

Comment:
I do like sentence case more than title case, but we need to be consistent throughout Hammer output. 

```suggestion
        collection :content_view_environments, _('Content View Environments') do
```